client/package.json

```diff
@@ -8,7 +8,7 @@
     "build": "esbuild ./target/scala-2.13/client-opt/main.js --bundle --outdir=../server/src/main/assets/ --minify"
   },
   "dependencies": {
-    "@drt/drt-react": "UKHomeOffice/drt-react#1.7.2",
+    "@drt/drt-react": "github:UKHomeOffice/drt-react#1.7.4",
```

Comment:
You don't need the `github:` prefix - see https://docs.npmjs.com/cli/v11/configuring-npm/package-json#github-urls